lib/contentful/meta_data.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Contentful.MetaData do
+  @moduledoc """
+  The MetaData represents internal additional data for Contetnful API objects, usually found in the 
+  "sys" part of the response objects. It's also referred to as "common properties".
+
+  See the [official documentation for more information](https://www.contentful.com/developers/docs/references/content-delivery-api/#/reference/locales).
+  """
+  defstruct [:id, :revision, :version]
```

Comment:
Linking is something I have to investigate, the old version of this library was more of an request wrapper, so resolving includes via links made sense.

For this version, since Elixir is mostly used on the backend, I'd want link resolution to be a conscious thing, e.g.

```elixir
{:ok, space} = Contentful.Delivery.Space.fetch_one
entries = 
  space
  |> Contenful.Delivery.Entries.stream()
  |> Stream.map(&Contentful.Assets.resolve/1)
  |> Enum.take(2)
```

I'd like to give the user more control over what they actually resolve.